process/testing/winfv/setup-fv-capz.sh

```diff
@@ -0,0 +1,163 @@
+  fi
+}
+
+function prepare_fv(){
+  make -C $CALICO_HOME/felix fv/win-fv.exe
+  make -C $CALICO_HOME/cni-plugin fv/win-fv.exe
+
+  FV_RUN_FELIX=$CALICO_HOME/process/testing/winfv/run-felix-fv.ps1
+  cp $CALICO_HOME/process/testing/winfv/run-fv-full.ps1 $FV_RUN_FELIX
+  sed -i "s?<your kube version>?${KUBE_VERSION}?g" $FV_RUN_FELIX
+  sed -i "s?<your linux pip>?${LINUX_NODE}?g" $FV_RUN_FELIX
+  sed -i "s?<your os version>?${WINDOWS_SERVER_VERSION}?g" $FV_RUN_FELIX
+  sed -i "s?<your container runtime>?containerd?g" $FV_RUN_FELIX
+  sed -i "s?<your containerd version>?${CONTAINERD_VERSION}?g" $FV_RUN_FELIX
+  sed -i "s?<your fv type>?tigera-felix?g" $FV_RUN_FELIX
+  sed -i "s?<your fv provisioner>?capz?g" $FV_RUN_FELIX
+  sed -i "s?win-fv.exe?win-felix-fv.exe?g" $FV_RUN_FELIX
+
+  FV_RUN_CNI=$CALICO_HOME/process/testing/winfv/run-cni-fv.ps1
+  cp $CALICO_HOME/process/testing/winfv/run-fv-cni-plugin.ps1 $FV_RUN_CNI
+  sed -i "s?<your kube version>?${KUBE_VERSION}?g" $FV_RUN_CNI
+  sed -i "s?<your linux pip>?${LINUX_NODE}?g" $FV_RUN_CNI
+  sed -i "s?<your os version>?${WINDOWS_SERVER_VERSION}?g" $FV_RUN_CNI
+  sed -i "s?<your container runtime>?containerd?g" $FV_RUN_CNI
+  sed -i "s?<your containerd version>?${CONTAINERD_VERSION}?g" $FV_RUN_CNI
+  sed -i "s?win-fv.exe?win-cni-fv.exe?g" $FV_RUN_CNI
+
+  upload_fv ${WIN_NODE_IP}
+}
+
+function wait_for_nodes(){
+  #Wait for calico-node-windows daemon set to update
+  sleep 30
+  for i in `seq 1 30`; do
+    if [[ `kubectl get ds calico-node-windows -n calico-system --no-headers --kubeconfig $KUBECONFIG | awk -v OFS='\t\t' '{print $6}'` = "$WIN_NODE_COUNT" ]] ; then
+      echo "Calico Node Windows is ready"
+      return
+    fi
+    echo "Waiting for Calico Node Windows to update"
+    sleep 30
+  done
+  echo "Node windows did not start"
+  exit 1
+}
+
+function update_windows_node(){
+  upload_calico_node
+  kubectl annotate ds -n calico-system calico-node-windows unsupported.operator.tigera.io/ignore="true" --kubeconfig $KUBECONFIG
+  kubectl patch ds -n calico-system calico-node-windows --patch-file $CALICO_HOME/process/testing/winfv/calico-node-windows.yaml  --kubeconfig $KUBECONFIG
+}
+
+function start_test_infra(){
+  $CALICO_HOME/process/testing/winfv/infra/setup.sh $KUBECONFIG
+
+  #Wait for porter pod to be running on windows node
+  for i in `seq 1 30`; do
+   if [[ `kubectl -n demo get pods porter --no-headers -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,PodIP:status.podIP,READY-true:status.containerStatuses[*].ready --kubeconfig $KUBECONFIG | awk -v OFS='\t\t' '{print $4}'` = "true" ]] ; then
+     echo "Porter is ready"
+     return
+   fi
+   echo "Waiting for porter to be ready"
+   sleep 30
+  done
+  echo "Porter windows did not start"
+  exit 1
+}
+
+function run_windows_fv(){
+  $CAPZ_LOCATION/ssh-node.sh $WIN_NODE_IP 'c:\\run-felix-fv.ps1' > output
+  $CAPZ_LOCATION/ssh-node.sh $WIN_NODE_IP 'c:\\run-cni-fv.ps1' > output
```

Comment:
The way the semaphore jobs are currently set up, we should conditionally run one of these tests, not both of them here (based on FV_TYPE I think)